Changes.md

```diff
@@ -1,5 +1,6 @@
 # v1.2.33 (unpublished, target date 15 February 2022)
 - no changes yet
+- rm $SRCDIR/sparql/owldef-violation.sparql $SRCDIR/sparql/nolabels-violation.sparql $SRCDIR/sparql/def-lacks-xref-violation.sparql $SRCDIR/sparql/obsolete-violation.sparql $SRCDIR/sparql/redundant-subClassOf-violation.sparql
```

Comment:
the variable wont work.. you should change to actual path

Reply:
done